### PR TITLE
Modify the pod spec by adding resource information

### PR DESCRIPTION
**Pull request by jshaofuturewei**
It is related to https://github.com/futurewei-cloud/global-resource-scheduler/issues/20

### CODE REVIEW

**Review by CoderKevinZhang — CHANGES_REQUESTED** [review]
I check your latest commit. Did you mixed your own change together with `make update`? We better separate them.

**Comment by jshaofuturewei** [comment]
> I check your latest commit. Did you mixed your own change together with `make update`? We better separate them.

I have divided the request into 3 commits.

File: pkg/kubelet/container/helpers_test.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
[the remark this answers is not in the document]
Comment (jshaofuturewei) [reply pkg/kubelet/container/helpers_test.go:26]: Other packages might have v1 version. Add this to clarify.

File: pkg/kubelet/container/helpers_test.go [hunk 2]
```diff
@@ -534,7 +534,7 @@ func TestHashContainer(t *testing.T) {
 				ResizePolicy: []v1.ResizePolicy{cpuPolicyRestart, memPolicyNoRestart},
 			},
 			false,
-			0xa76b1795,
+			0x8f232301,
```
[the remark this answers is not in the document]
Comment (jshaofuturewei) [reply pkg/kubelet/container/helpers_test.go:537]: Yes, just tracking its address.

**Review by CoderKevinZhang — APPROVED** [review]

**Review by kimeunju108 — APPROVED** [review]
LGTM

**Pull request merged**